fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -727,6 +728,9 @@ private void initAggregateBuiltins() {
                 Lists.newArrayList(Type.VARCHAR, Type.VARCHAR), Type.VARCHAR, Type.VARCHAR,
                 false, false, false));
 
+        addBuiltin(AggregateFunction.createBuiltin(WINDOW_FUNNEL, Lists.newArrayList(Type.BIGINT, Type.DATETIME, Type.ARRAY_BOOLEAN),
```

Comment:
```suggestion
        addBuiltin(AggregateFunction.createBuiltin(WINDOW_FUNNEL, Lists.newArrayList(Type.BIGINT, Type.DATETIME, Type.INT, Type.ARRAY_BOOLEAN),
```
Can we add a 'mode' parameter to windwo_funnel? so we can provide differenct mode at future.